api.planx.uk/modules/analytics/metabase/collection/service.ts

```diff
@@ -0,0 +1,35 @@
+import { updateMetabaseId } from "./updateMetabaseId.js";
+import type { NewCollectionParams } from "./types.js";
+import { getTeamIdAndMetabaseId } from "./getTeamIdAndMetabaseId.js";
+import { createCollection } from "./createCollection.js";
+
+/**
+ * The `getTeamIdAndMetabaseId()` function is run here to first get teams.id and .metabase_id from PlanX db, if present,
+ * so that the service can figure out if it needs to run `createCollection()` or not.
+ * Instead of running `getTeamIdAndMetabaseId()` first in the controller, it is encapsulated in `createTeamCollection` to keep business logic in `service.ts` instead of `controller.ts`.
+ */
```

Comment:
I actually think the code here is pretty clear and easy to follow - we can safely remove this comment. I found it a little harder to follow and reason with in English than in code 😂